kore/test/Test/Kore/Builtin/Builtin.hs

```diff
@@ -92,7 +92,7 @@ import Kore.Simplify.InjSimplifier
 import Kore.Simplify.Not qualified as Not
 import Kore.Simplify.OverloadSimplifier
 import Kore.Simplify.Pattern qualified as Pattern
-import Kore.Simplify.Simplify hiding (simplifyPattern)
+import Kore.Simplify.Simplify hiding (runSimplifier, simplifyPattern)
```

Comment:
nit) Not really happy with this. Having a different `runSimplifier` in scope is very confusing (it was already confusing before), maybe we should rename the one in `Test.Kore.Simplifier` while we are at it.